.github/workflows/set-rollout.yaml

```diff
@@ -0,0 +1,50 @@
+name: Set rollout by path
+
+on: [deployment_status]
+
+jobs:
+  set-rollout:
+    if: ${{ github.event.deployment.task == 'upload-to-cdn' && github.event.deployment_status.state == 'success' }}
+    runs-on: ubuntu-latest
+    steps:
+      - uses: hmarr/debug-action@v2
+
+      # Dev
+      - name: Set Rollout - Development
+        uses: decentraland/set-rollout-action@v2
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
+
+          # Repo deployment info
+          ref: ${{ github.event.deployment.ref }}
+          sha: ${{ github.event.deployment.sha }}
+
+          # CDN information
+          packageName: ${{ github.event.deployment.payload.packageName }}
+          packageVersion: ${{ github.event.deployment.payload.packageVersion }}
+
+          # Rollout information
+          deploymentPath: 'vesting-dashboard'
+          deploymentEnvironment: 'zone'
+          deploymentName: '_site'
+          percentage: 100
+
+      # Stg
+      - name: Set Rollout - Staging
```

Comment:
This job deploys the site to staging, but you're not enabling the staging environment in the definitions repo, would you mind removing it?